### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ThomasNifong**
Less bold
Edit headers
-98 words

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,54 +1,17 @@
-### What Are We Fighting For?
```
Comment (beaurancourt) [inline docs/index.md:1]: I liked the lighter, narrative tone

File: docs/index.md [hunk 2]
```diff
@@ -1,54 +1,17 @@
...
-As 5e players, we find ourselves making choices that affect combat. Either a roleplay choice will force or avoid combat.
-Perhaps a build choice increases damage at the cost of survivability.
-Maybe you need to need to decide between casting a damage spell, a buffing spell, a control spell, or a healing spell.
-An enemy is about to attack you for sure, should you fight defensively, or attack it on your turn?
+### What should a player do in combat?
+In 5e D&D, players' choices are all related to combat in some way. A roleplay choice will force or avoid combat. Choosing class features can improve damage, survivability, allow a character to take more actions or avoid combat, and usually each one comes at the cost of another. In combat, you need to decide between casting a spell that deals damage, heals or changes your allies' or enemies' conditions. An enemy is about to attack you for sure, should you dodge, disengage or attack? If there's an objective function, you can test each option and know which choice is right.
```
Comment (beaurancourt) [inline docs/index.md:2]: The intro sentence goes from the inclusive / friendly "we find ourselves" to a more abstract "the player" - do you think that's better?

Comment (beaurancourt) [reply docs/index.md:2]: The second one is definitely more of a wikipedia style, but wasn't what I was going for here

File: docs/index.md [hunk 3]
```diff
@@ -1,54 +1,17 @@
...
 
-Here, I propose the an optimization function: maximize the probability that your party lives through an adventuring day as defined by page 84 of the DMG.
-One such adventuring day is 6 medium difficulty encounters for a party of 4 PCs at level 5.
-The DMG suggests a short rest 1/3 and 2/3 the way through an adventuring day.
+Damage, control and defense are all measurable individually with the rules, but there isn't a way to compare how each one stacks up at the cost of the others. If you have great defenses and healing, then your enemies have a harder time killing the party. If you do great damage, then you get through your fights faster, and your enemies have a harder time killing the party. If you have great crowd control, then you can use conditions to delay or prevent enemy attacks, and your enemies have a harder time killing the party. The problem is not knowing which  gives your enemies the hardest time killing the party relative to the others. The goal of the objective function for making decisions should track the remaining resources at the end of an encounter.
```
Comment (beaurancourt) [inline docs/index.md:10]: > Damage, control and defense are all measurable individually with the rules, but there isn't a way to compare how each one stacks up at the cost of the others.

But there is a way

File: docs/index.md [hunk 4]
```diff
@@ -1,54 +1,17 @@
...
 
-If you do great damage, then awesome! You got through your fights faster, and **your enemies have a harder time killing the party.**
-
-If you have great crowd control, then awesome! You got through your fights with more control, and **your enemies have a harder time killing the party.**
-
-If you have high utility, then awesome! You were able to cleverly bypass combat, or swing the battlefield to your advantage, and **your enemies have a harder time killing the party.**
-
-If we have to give up utility for crowd control, how do we decide? If we have to give up crowd control for damage, how do we decide?
-
-**You decide by doing the thing that maximizes the chance that your party lives to fight another day!**
```
Comment (beaurancourt) [inline docs/index.md:31]: What was wrong with the bolding? I thought it helped hammer home the point that what mappers is not dying

File: docs/index.md (hunk 1 above)
Comment (ThomasNifong) [reply docs/index.md:1]: https://www.youtube.com/watch?v=9OVv-J-LXQU

That, and I associate "fighting" with something related to politics or cancer. It needs to refer to the mechanics of D&D. "What are we doing in D&D combat" is the message people need to get from the first header.

File: docs/index.md (hunk 2 above)
Comment (ThomasNifong) [reply docs/index.md:2]: You could say "we", "you" or any other colloquialism. I wouldn't use "we find ourselves." I reserve that for when it's true in a geographic sense, ie. "After being lost we found ourselves in a clearing."

File: docs/index.md (hunk 3 above)
Comment (ThomasNifong) [reply docs/index.md:10]: Correct. I should have specified that there isn't a way to measure them in the core rule books that is comparable to what we are doing.

File: docs/index.md (hunk 4 above)
Comment (ThomasNifong) [reply docs/index.md:31]: The line break and bold is probably fine. I would reserve bolding for keywords. If you want to bold "your enemies have a harder time killing the party," I would still remove bold from the other lines at least until you get to the next header.

File: docs/index.md (hunk 1 above)
Comment (beaurancourt) [reply docs/index.md:1]: It was meant as a double entendre - both as a melodramatic, moral hook "why do we fight?", and as a poke at objective functions.

**Comment by beaurancourt** [comment]
I think that I want to keep the general tone in tact, so feedback with that in mind would be more helpful

**Comment by beaurancourt** [comment]
Closing this PR, as I added a bunch to the page

**Pull request closed without merging**